roles/openshift_health_checker/test/openshift_check_test.py

```diff
@@ -38,3 +38,24 @@ def test_get_var_error(task_vars, missing_keys):
 def test_get_var_default(task_vars, missing_keys):
     default = object()
     assert get_var(task_vars, *missing_keys, default=default) == default
+
+
+@pytest.mark.parametrize("release,expected", [
+    ('1.2', '1.2'),
+    (1.2, '1.2'),
+    ('v1.2', '1.2'),
+    ('v1.2.3.4', '1.2.3.4'),
```

Comment:
- These two tests could probably have the same list of inputs (with different expected values, of course). They are now off by one.

- Please add a test for a base case: empty input.

- Please add test(s) for bad input.